servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -134,6 +134,14 @@ public void tearDown() {
     api.close();
   }
 
+  @Test
+  void createRecreateDefaultBranch() throws NessieConflictException, NessieNotFoundException {
+    api.deleteBranch().branch(api.getDefaultBranch()).delete();
+
+    api.createReference().reference(Branch.of("main", null)).create();
+    api.getReference().refName("main").get();
```

Comment:
We don't need any assertions for this test?

Reply:
It shall just pass

Reply:
maybe just add a `assertThat(api.getReference().refName("main").get()).isNotNull()` check

Reply:
which will never return null but throw an exception, right?